context/app/static/js/hooks/useSearchHits.js

```diff
@@ -0,0 +1,33 @@
+import { useState, useEffect } from 'react';
+import { getAuthHeader } from 'js/helpers/functions';
+
+function useSearchHits(query, elasticsearchEndpoint, nexusToken) {
+  const [searchData, setSearchData] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
Is `isLoading` actually used? I think it can be removed.

Reply:
I'll likely use this when working on the layout shift for the table of contents sidebar.